plugin.program.chrome.launcher/default.py

```diff
@@ -0,0 +1,382 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import urllib
+import sys
+import re
+import os
+import time
+import subprocess
+import xbmcplugin
+import xbmcgui
+import xbmcaddon
+
+
+addon = xbmcaddon.Addon()
+pluginhandle = int(sys.argv[1])
+addonID = addon.getAddonInfo('id')
+addonPath = addon.getAddonInfo('path')
+translation = addon.getLocalizedString
+osWin = xbmc.getCondVisibility('system.platform.windows')
```

Comment:
xbmc.getCondVisibility() seems to have disappeared from the Jarvis documentation
http://mirrors.kodi.tv/docs/python-docs/16.x-jarvis/xbmc.html

@phil65 is this deprecated or just missing from the Jarvis doc?